API/Models/SeedData.cs

```diff
@@ -0,0 +1,226 @@
+                    MainCameraScore = 7.8f,
+                    SelfieCameraScore = 7.1f,
+                    StorageScore = 8.5f,
+                    BatteryLifeScore = 6.8f,
+                    DurabilityScore = 8.0f,
+                    PopularityScore = 8.9f,
+                    ScreenSizeScore = 8.8f,
+                    PriceScore = 7.5f
+                }
+            );
+            #endregion
+
+            #region QuestionSeed
+            _context.Questions.AddRange(
+                new Question
+                {
+                    ID = 1,
+                    QuestionContent = "How much are you willing to spend?",
+                    Aspect = Aspect.Price
+                },
+                new Question
+                {
+                    ID = 2,
+                    QuestionContent = "Do you want to use you phone for photography?",
+                    Aspect = Aspect.MainCamera
+                },
+                new Question
+                {
+                    ID = 3,
+                    QuestionContent = "Do you want to play video games on your phone?",
+                    Aspect = Aspect.ProcessingPower
+                },
+                new Question
+                {
+                    ID = 4,
+                    QuestionContent = "Will your phone be exposed to harsh environments?",
+                    Aspect = Aspect.Durability
+                },
+                new Question
+                {
+                    ID = 5,
+                    QuestionContent = "Are you planning to stroe lots of files?",
+                    Aspect = Aspect.Storage
+                }
+            );
+            #endregion
+
+            #region QuestionOptionSeed
+            _context.QuestionOptions.AddRange(
+                new QuestionOption
+                {
+                    ID = 1,
+                    QuestionID = 1,
+                    OptionContent = "range",
+                    QuestionMultiplier = 0.5f
+                },
+                new QuestionOption
+                {
+                    ID = 2,
+                    QuestionID = 2,
+                    OptionContent = "stars",
+                    QuestionMultiplier = 0.5f
+                },
+                new QuestionOption
+                {
+                    ID = 3,
+                    QuestionID = 3,
+                    OptionContent = "stars",
+                    QuestionMultiplier = 0.5f
+                },
+                new QuestionOption
+                {
+                    ID = 4,
+                    QuestionID = 4,
+                    OptionContent = "stars",
+                    QuestionMultiplier = 0.5f
+                },
+                new QuestionOption
+                {
+                    ID = 5,
+                    QuestionID = 5,
+                    OptionContent = "stars",
+                    QuestionMultiplier = 0.5f
+                }
+            );
+            #endregion
```

Comment:
Why this data here is hardcoded?

Reply:
Seeders could be randomized and not hardcoded. As well as have their own class for each table

Reply:
As I've mentioned in the description, this is mockup data meant to facilitate testing.

Reply:
Ok. Wouldn't it be better to create factories for this purpose? Also, this class is wayy too big, consider moving put this code to separate classes as @lukasrakauskas suggested.

Reply:
@wizacass I'd love to have laravel-like model factories here too, but after doing some search, I was not able to find anything similar. If you have any information, do share. For now, I just separated regions into classes for each model.

Reply:
Ok. So for now let's keep it as it is and let's do some research how we can improve on this. Generally, data shouldn't be hardcoded as it is now.